src/tests/system/tests/test_infopipe.py

```diff
@@ -278,3 +279,193 @@ def test_infopipe__lookup_user_with_extra_attributes(client: Client, provider: G
 
     result = client.sssd.svc.status("sssd")
     assert result.rc == 0, "Service is not running!"
+
+
+@pytest.mark.importance("medium")
+@pytest.mark.topology(KnownTopology.LDAP)
+def test_infopipe__get_group_properties(client: Client, provider: GenericProvider):
+    """
+    :title: Get a group's properties via Infopipe
+    :setup:
+        1. Add users 'user1' and 'user2'
+        2. Add group 'group-1' with 'user1' and 'user2' as members
+        3. Start SSSD
+    :steps:
+        1. Call FindByName() to get the object path for 'group-1'.
+        2. Create a group object from the retrieved path.
+        3. Get the GID property using Get().
+        4. Get all properties using GetAll() and verify the member list.
+        5. Validate core properties (name, GID, users).
+    :expectedresults:
+        1. Group 'group-1' is found and returns the correct object path.
+        2. Group object is successfully created.
+        3. GID property is 30001.
+        4. Member list contains object paths for 'user1' and 'user2'.
+        5. Properties match: name is 'group-1', GID is 30001, and users list has 2 members.
+    :customerscenario: False
+    """
+    # Setup
+    u1 = provider.user("user1").add(uid=10001)
+    u2 = provider.user("user2").add(uid=10002)
+    provider.group("group-1").add(gid=30001).add_members([u1, u2])
+    client.sssd.start()
+
+    # Verify users via Infopipe
+    users = client.ifp.getObject("/org/freedesktop/sssd/infopipe/Users")
+    user1_path = users.FindByName("user1")
+    user2_path = users.FindByName("user2")
```

Comment:
Use single `UpdateMemberList` group interface method, instead of 'FindByName' for every member.

Reply:
(Pdb) group.UpdateMemberList()
*** IndexError: list index out of range
(Pdb)

Reply:
https://github.com/SSSD/sssd/blob/ad7dc210f79e9b521ff26449d10a0348debff4a8/src/responder/ifp/ifp_iface/ifp_iface.xml#L245

Reply:
It is working but giving error :)  

(Pdb)     props = group.GetAll("org.freedesktop.sssd.infopipe.Groups.Group")
(Pdb)     print(f"Group properties: {props}")
Group properties: {'name': 'group-1', 'gidNumber': 30001, 'users': [], 'groups': []}
(Pdb) group.UpdateMemberList()
*** IndexError: list index out of range
(Pdb)     props = group.GetAll("org.freedesktop.sssd.infopipe.Groups.Group")
(Pdb)     print(f"Group properties: {props}")
Group properties: {'name': 'group-1', 'gidNumber': 30001, 'users': ['/org/freedesktop/sssd/infopipe/Users/test/10001', '/org/freedesktop/sssd/infopipe/Users/test/10002'], 'groups': []}
(Pdb)

Reply:
> (Pdb) group.UpdateMemberList()
> *** IndexError: list index out of range

@aplopez, why is this?